torch/_decomp/__init__.py

```diff
@@ -358,7 +358,8 @@ def core_aten_decompositions() -> Dict[torch._ops.OperatorBase, Callable]:
             aten.special_entr,
             aten.special_log_ndtr,
             aten.special_xlog1py,
-            aten.std.correction,
+            aten.std,
+            aten.std_mean,
```

Comment:
Can you also register the `std_mean` decomposition for all the `std_mean` variants, rather than only the correction one?